readthedocs/profiles/views.py

```diff
@@ -28,6 +34,7 @@
 from readthedocs.core.mixins import PrivateViewMixin
 from readthedocs.core.models import UserProfile
 from readthedocs.core.utils.extend import SettingsOverrideObject
+from readthedocs.projects.utils import get_csv_file
```

Comment:
Took this from https://github.com/readthedocs/readthedocs.org/pull/8498/files#diff-328457e4e9f836c833e749d11542d3647fa7f60e752aa6b4ca108a1714725a7f